### PR TITLE
Player queue implementation

### PR DESCRIPTION
**Pull request by NDacevic**
Player plays from queue.

Added styling

### CODE REVIEW

File: src/components/Player.vue [hunk 1]
```diff
@@ -168,7 +183,7 @@ export default {
       updateTrackInterval: Object,
       volume: 100,
       playing: false,
-      muted:false
+      muted: false,
```
Comment (Martin-Ekman) [inline src/components/Player.vue:186]: Trailing comma!

File: src/store/store.js [hunk 2]
```diff
@@ -29,6 +29,9 @@ export default new Vuex.Store({
     setSearchHasBeenPerformed(state, searchHasBeenPerformed) {
       state.searchHasBeenPerformed = searchHasBeenPerformed;
     },
+    removeTopFromQueue(state){
```
Comment (Martin-Ekman) [inline src/store/store.js:35]: Formattering

File: src/store/store.js [hunk 3]
```diff
@@ -78,6 +81,9 @@ export default new Vuex.Store({
     },
     getSearchHasBeenPerformed(state) {
       return state.searchHasBeenPerformed;
+    },
+    getQueuedTracksLength(state){
```
Comment (Martin-Ekman) [inline src/store/store.js]: Formattering

File: src/store/store.js [hunk 4]
```diff
@@ -78,6 +81,9 @@ export default new Vuex.Store({
...
+      return state.queuedTracks.length
```
Comment (Martin-Ekman) [inline src/store/store.js]: semi-kolon!

File: src/styles/queueStyles.scss [hunk 5]
```diff
@@ -63,13 +63,29 @@
     &.md-theme-default.md-raised:not([disabled]).md-accent {
       background-color: transparent;
     }
+
+    &-content{
```
Comment (Martin-Ekman) [inline src/styles/queueStyles.scss:67]: Formattering!

**Review by mikaelollhage — DISMISSED** [review]

File: src/styles/styles.scss [hunk 6]
```diff
@@ -5,11 +5,17 @@
   font-weight: 500;
   background-color: $color-primary-4;
   color: $mainFontColor;
+  box-sizing: border-box;
 }
+
+#app{
```
Comment (Martin-Ekman) [inline src/styles/styles.scss:11]: Ajabaja

**Review by Martin-Ekman — DISMISSED** [review]
Lite smått med formattering :D Great work!

**Review by mikaelollhage — DISMISSED** [review]

**Review by mikaelollhage — APPROVED** [review]

**Pull request closed without merging**